src/test/java/org/cactoos/list/ListEnvelopeTest.java

```diff
@@ -81,9 +77,46 @@ public void returnsListIteratorWithUnsupportedAdd() {
                 inner.add("ten");
                 return inner;
             }
-        ) { };
+        ) {
+        };
         final ListIterator<String> iterator = list.listIterator();
         iterator.next();
         iterator.add("twenty");
     }
+
+    @Test
+    public void getsPreviousIndex() {
+        MatcherAssert.assertThat(
+            "List iterator returns incorrect previous index",
+            new ListOf<>(1).listIterator().previousIndex(),
+            new IsEqual<>(-1)
+        );
+    }
+
+    @Test
+    public void getsPrevious() {
+        MatcherAssert.assertThat(
+            "List iterator returns incorrect previous item",
+            new ListOf<>(3, 7).listIterator(1).previous(),
+            new IsEqual<>(3)
+        );
```

Comment:
@proshin-roman See above, about `ListIterator(List, int)` constructor; if implemented, could be used here.